_opensearch/snapshots/snapshot-management.md

```diff
@@ -0,0 +1,78 @@
+
+An SM schedule is a custom [cron]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/cron) expression. It consists of two parts: a creation schedule, and a deletion schedule. You must set up a schedule that specifies the frequency and timing of taking snapshots. Optionally, you can set up a separate schedule for deleting snapshots.
+
+An SM configuration includes the indices and repository for the snapshots, and supports all parameters you can define when [creating a snapshot]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#take-snapshots) using the API. Additionally, you can specify the format and time zone for the date used in the snapshot's name.
+
+
+## Performance 
+
+One snapshot can contain as many indices as there are in the cluster. We expect at most dozens of SM policies in one cluster, but a snapshot repository can safely scale to thousands of snapshots. However, to manage its metadata, a large repository requires more memory on the cluster manager node. 
+
+Snapshot management depends on the job scheduler plugin to schedule a job that is run periodically. Each SM policy corresponds to one SM scheduled job. The scheduled job is lightweight, so the burden of SM depends on the snapshot creation frequency and the burden of running the snapshot operation itself. 
+
+## Concurrency 
+
+An SM policy does not support concurrent snapshot operations, since too many such operations may degrade the cluster. Snapshot operations (creation or deletion) are performed asynchronously: SM does not start a new operation until the previous operation finishes.
+
+We don't recommend creating several SM policies with the same schedule and overlapping indices in one cluster, because it leads to concurrent snapshot creation on same indices and hinders performance. 
+{: .warning }
+
+
+We don't recommend setting up the same repository for multiple SM policies with same schedule in different clusters, since it may bring a sudden spike of burden on this repository.
+{: .warning }
+
+## Failure management
+
+If a snapshot operation fails, it is retried for a maximum of three times. The failure message is saved in metadata and is overwritten if a subsequent snapshot operation succeeds. You can view the failure message using the [explain API]({{site.url}}{{site.baseurl}}/opensearch/snapshots/sm-api#explain). When using Dashboards, you can view the failure message in the [policy details page]({{site.url}}{{site.baseurl}}/opensearch/snapshots/sm-dashboards#view-edit-or-delete-an-sm-policy). Possible reasons for failure include red index status, and shard reallocation.
```

Comment:
The failure message is saved in `metadata.latest_execution` and is overwritten when the subsequent snapshot operation starts.